openmls_rust_crypto/src/provider.rs

```diff
@@ -409,4 +409,4 @@ impl OpenMlsRand for RustCrypto {
             .map_err(|_| "Unable to collect enough randomness.")?;
         Ok(out)
     }
-}
+}
```

Comment:
Shouldn't there be a newline here?

Reply:
hm, I just moved it...  not sure what happened